docs/readmes/proposals/p027_magma_outbound_roaming_support.md

```diff
@@ -0,0 +1,136 @@
+# Outbound Roaming
+
+Link to discussed detailed doc <https://docs.google.com/document/d/17OYrYw6PMxFEWi_Ubx8eD1Qt3UT2VbchR_FIT5ivxI0>
+
+## 1 Objectives
+
+The objective of this work is to add support in Magma for deployments where outbound roaming is required. Specifically, adding support to expose the server sides of s6a and s8 to roaming partner core networks. This will enable outbound roaming to be supported in a 3GPP compliant and standard way which operators are used to.
+Software built to accomplish this will be open source under BSD-3 license and will be committed to the Magma software repository under the governance of the Linux foundation, such that it can be effectively maintained in the future releases.
+
+## 2 Background
+
+### 2.1: Terminology
+
+**S6a** refers to the Authentication, Location, and Service definition interface of an HSS in a mobile core.
+**S8** refers to the control and user plane interfaces used when user data is home routed during an outbound roaming session.
+
+### 2.2 Outbound roaming
+
+Outbound roaming is when a subscriber gains access to services from a visited operator (VPLMN) other than their home network operator (HPLMN). In this case the HPLMN is a Magma deployment, and the VPLMN is another network operator with which this HPLMN has a roaming relationship.
+
+When outbound roaming, the standard interfaces are s6a and s8 which allow the VPLMN to authenticate the SIM credentials with the home network and route user traffic back to the home network. See diagram below from 3GPP TS 23.401 V17.3.0 (2021-12).
+
+![arch](https://user-images.githubusercontent.com/93994458/154709583-14f9bb52-486b-4908-b250-e521627b6f04.png)
+
+Currently, Magma supports only inbound roaming where magma is the VPLMN and another operator/core is the HPLMN. Magma supports the client side of both S6a and S8 interfaces.
+
+By implementing the server side of S6a and S8, this project will add support for outbound roaming to Magma.
+
+## 3 Implementation
+
+### 3.1 Diagrams
+
+The Roaming Gateway diagram shows the services which will run in the Roaming gateway. These services will support the user plane data transport for both inbound roaming GTP aggregation and outbound roaming when traffic is home routed.
+
+![roaming gateway](https://user-images.githubusercontent.com/93994458/154709967-7cd453ba-2070-4733-9e3f-dcc6f38624d2.png)
```

Comment:
[Mermaid diagrams](https://mermaid-js.github.io/mermaid/) should be preferred over binary blob includes. [They work in Github and DocuSaurus](https://github.com/magma/magma/blob/master/docs/readmes/feg/s1ap_federated_test.md).